scipy/stats/tests/test_stats.py

```diff
@@ -3698,8 +3725,9 @@ def test_skew_propagate_nan(self, xp):
 
     def test_skew_constant_value(self, xp):
         # Skewness of a constant input should be NaN (gh-16061)
-        with pytest.warns(RuntimeWarning, match="Precision loss occurred"):
-            a = xp.asarray([-0.27829495]*10)  # xp.repeat not currently available
+        a = xp.asarray([-0.27829495]*10)  # xp.repeat not currently available
+
+        with eager_warns(a, RuntimeWarning, match="Precision loss occurred"):
```

Comment:
Hmm, I see that only *one* of these has to warn for the test to pass. Looks like that's not what we want. Just thought I'd mention it, but no need to fix it here, since it's not new with this PR.